modules/ve-mw/preinit/styles/ve.init.mw.DesktopArticleTarget.init.css

```diff
@@ -76,6 +76,9 @@
 .ve-activated .ve-init-mw-desktopArticleTarget-originalContent #catlinks a {
 	opacity: 1;
 }
+.mw-ext-cite-error {
+	display: inline !important;
```

Comment:
To minimize changes to this submodule let's do this in FandomVisualEditor extension instead.